src/lib/components/ProductForm.svelte

```diff
@@ -629,20 +664,24 @@
 				type="checkbox"
 				bind:checked={product.hasVariations}
 				name="hasVariations"
-				disabled={!product.standalone}
 			/>
 			Product has light variations (no stock difference)
 		</label>
 		{#if product.hasVariations}
 			{#each [...(product.variations || []), ...Array(variationLines).fill( { name: '', value: '', price: 0 } )].slice(0, variationLines) as variation, i}
 				<div class="flex gap-4">
 					{#if variation.name && variation.value}
+						<button
+							class={i === 0 ? 'mt-8' : ''}
```

Comment:
Why put an extra margin specifically in the first element of any branch?
Wouldn't it be easier to read if it were set in the parent container?